### PR TITLE
Support for entity categories, configuration URL and other 2021.11 related features

### PR DESCRIPTION
**Pull request by iMicknl**
Cannot be merged until next HA release

<a href="https://gitpod.io/#https://github.com/iMicknl/ha-tahoma/pull/611"></a>

Screen Shot 2021-10-29 at 16 30 10

Screen Shot 2021-10-29 at 16 31 10

### CODE REVIEW

**Comment by iMicknl** [comment]
> You will have to increase the required HA version I guess.

Good one, will do :-). I marked it as a draft for that sense. Will add the key soon.

File: custom_components/tahoma/entity.py [hunk 1]
```diff
@@ -98,6 +98,7 @@ def device_info(self) -> DeviceInfo:
             sw_version=self.executor.select_attribute(CORE_FIRMWARE_REVISION),
             suggested_area=self.coordinator.areas[self.device.place_oid],
             via_device=self.executor.get_gateway_id(),
+            configuration_url=self.coordinator.client.server.configuration_url,
```
Comment (tetienne) [inline custom_components/tahoma/entity.py:101]: Apparently, there is no effect on my env. I cannot see any url in my device pages.

Comment (iMicknl) [reply custom_components/tahoma/entity.py:101]: Are you sure you have the newest update + front-end reloaded?

Comment (iMicknl) [reply custom_components/tahoma/entity.py:101]: Screen Shot 2021-11-03 at 21 51 04

Comment (tetienne) [reply custom_components/tahoma/entity.py:101]: Ah yes,  I was looking for the wrong thing.

**Review by tetienne — APPROVED** [review]

**Pull request merged**